website/docs/client/client-serialization.mdx

```diff
@@ -366,7 +366,7 @@ graphql {
 By default, `ServiceLoader` mechanism will load the first available GraphQL client serializer from the classpath.
 
 ```kotlin
-val client = GraphQLWebClient(
+val client = GraphQLKtorClient(
   url = "http://localhost:8080/graphql"
```

Comment:
Nice catch (yeah it looks like copy and paste)! Can you also fix the constructor (Spring one takes String url vs URL object on Ktor)? i.e 

```kotlin
val client = GraphQLKtorClient(
  url = URL("http://localhost:8080/graphql")
```

I'm thinking for 5.0 we should update constructors on those clients to just accept a String...

Reply:
I did not see that, but amended it now :)